src/Parser.php

```diff
@@ -44,9 +44,9 @@ public static function parseDictionary(string $string): \stdClass
         return $value;
     }
 
-    public static function parseList(string $string): array
+    public static function parseList(string $string): OuterList
```

Comment:
I _think_ `OuterList` implementing `ArrayAccess` and `Traversable` interfaces should prevent this type change from causing issues